utils/cliutils/utils.go

```diff
@@ -24,12 +24,20 @@ type OnError string
 func init() {
 	// Initialize cli-core values.
 	cliUserAgent := os.Getenv(UserAgent)
-	if cliUserAgent == "" {
-		cliUserAgent = ClientAgent + "/" + CliVersion
+	if cliUserAgent != "" {
```

Comment:
Let's move the code that splits and sets the user agent name and version, into a different function, and then add a unit test for it (in utils_test.go).